styles/styles.css

```diff
@@ -21,6 +21,7 @@
   --clr-light-blue: rgb(65 121 179);
   --clr-green: rgb(128 188 0);
   --clr-white: rgb(255 255 255);
+  --clr-grey: rgb(240 243 244);
```

Comment:
just a note:

I don't necessarily think every color used anywhere on the site needs to be declared here, sorry for any confusion on that.

If it's used in multiple places, and especially if it's used for common elements, then it *must* be here and referenced as a variable. but there are quite a few one off colors used, and for those, I think it's fine to just inline them. If you want to put the one offs here, just to make it easier, that is fine with me, but I don't consider it required.

Use your best judgement obviously.